boysen_a06b.js

```diff
@@ -0,0 +1,100 @@
+/*
+Bill Boysen
+boysen_a06b.js
+INFO_2124_WW
+Thoendel
+1/26/2020
+*/
+
+const standardInput = process.stdin;
+standardInput.setEncoding('utf-8');
+
+const story = 
+    `You're walking along Dodge Street minding your own business
+and your suddenly run into the Wizard of Benson standing
+in on the corner of 50th and Dodge.
+
+Do you (C)ontinue walking? (S)top and ask for some magical help?
+(T)rip and fall into a pot hole?
+`;
+
+clearScreen();
+console.log(story);
+
+standardInput.on('data', function(text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    /* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+
+    //Conditional statement to choose output
+    //Executes if input is precisely "C"
+    if (text === "C") {

```

Comment:
This is a good check for the capital C - what if they type a lowercase c? Part of the assignment was to use a logical operator to check for the lowercase as well.
-3